src/diagnosers/21-web.py

```diff
@@ -185,7 +186,7 @@ def ipv6_is_important_for_this_domain():
                     )
                     AAAA_status = dnsrecords.get("data", {}).get("AAAA:@")
 
-                    return AAAA_status in ["OK", "WRONG"]
+                    return AAAA_status in ["OK", "WRONG"] or settings_get("misc.network.dns_exposure") != "ipv4"
```

Comment:
```suggestion
                    return AAAA_status in ["OK", "WRONG"] or settings_get("misc.network.dns_exposure") in ["both", "ipv6"]
```

Reply:
(I'm still unsure what we want to test here because my brain overflows :thinking: